### ISSUE TITLE
Specific train/test splits

### ISSUE THREAD

**Issue by jaberg** [hyperopt/hyperopt-sklearn#14]
Don't always split the train set 80-20, allow the caller to specify a train and validation set.
